cmd/webhook/admission/testdata/ingress/invalid-filters-and-predicates.json

```diff
@@ -0,0 +1,40 @@
+{
+    "request": {
+        "uid": "req-uid",
+        "name": "req1",
+        "namespace": "n1",
+        "object": {
+            "metadata": {
+                "name": "ing1",
+                "namespace": "ing1",
+                "annotations": {
+                    "zalando.org/skipper-filter": "status&(10) -> inlineContent(\"This should work\")",
```

Comment:
Is `status&(10)` by intention with `&`? If so why?

Reply:
I think we should use something more explicit as a wring eskip, e.g. `this-is-invalid(filter)`